webapp/src/plugin.jsx

```diff
@@ -18,6 +18,14 @@ export default class Plugin {
 
             registry.registerRootComponent(CreateIssueModal);
             registry.registerPostDropdownMenuComponent(CreateIssuePostMenuAction);
+
+            registry.registerWebSocketEventHandler(
+                'custom_' + PluginId + '_create_issue',
+                (payload) => {
```

Comment:
Not sure this is the right way to activate a modal client side. Will this not activate the create issue modal for all clients for that user?

I propose we cut this from jira 2.0 and leave it for 2.1. That way we can extend the plugin system to allow for client-side slash command interception and use that here.

Reply:
So... it /does/ activate the modal in other clients. I guess that's a breaking problem? @levb @jasonblais? 
I can expand this ticket to include making a `handleSlashCommandWillBeSent` hook, if you want to include this in v2.0?

Reply:
@crspeller @cpoile Given the issue at hand (opening a create issue modal on every client), agree: let's cut from v2.0, add the `handleSlashCommandWillBeSent` via [this ticket](https://mattermost.atlassian.net/browse/MM-15531), and then merge the command for v2.1.